README.md

```diff
@@ -320,6 +334,42 @@ All macros are now in UPPER CASE. Some macros (request, creative, etc.) have to
 Prior to the v0.3.0, there was only one Handlebars engine provided in the extra package.
 
 With the 0.3.0+, there are now 2 Handlebars engine:
+
 - HandlebarsEngine: to be used when building an AdRenderer without recommendations
 - RecommendationsHandlebarsEngine: to be used when building a 'recommendation' Ad Renderer
 
+### StatsClient helper
+
+You can add a StatsClient to your plugins, by importing helpers. Global tags with relevant datas such as artifact_id, build_id or version_id will be added automatically. When initiating the StatsD client, 4 options can be passed:
+
+- mandatory
+  - pluginType
+  - id (feed_id, channel_id, custom_action_id, or ad_renderer_id)
+- optional
+  - timerInMs (interval to send stats to datadog in ms (default = 10 minutes))
+  - logger
+
+```js
+this.statsClient = helpers.StatsClient.init({
+	pluginType: PluginType.AUDIENCE_FEED,
+	id: '1234',
+	logger: this.logger,
+});
+```
+
+Using StatsD, the StatsClient, can aggregate and send your stats to services such as Datadog. Increment your stats be calling addOrUpdateMetrics method.
+
+```js
+this.statsClient.addOrUpdateMetrics({
+	metrics: {
+		processed_users: { type: MetricsType.GAUGE, value: 4, tags: { datamart_id: '4521' } },
+		users_with_mobile_id_count: { type: MetricsType.GAUGE, value: 1, tags: { datamart_id: '4521' } },
+	},
+});
+this.statsClient.addOrUpdateMetrics({
+	metrics: {
+		processed_users: { type: MetricsType.GAUGE, value: 10, tags: { datamart_id: '4521' } },
+	},
+});
+this.statsClient.addOrUpdateMetrics({ metrics: { apiCallsError: { type: MetricsType.GAUGE, value: 10, tags: { statusCode: '500' } } } });
```

Comment:
@sbordeyne If you want to add some guidelines here based on the Excel document you've built ?